src/Data/Nat/Primality.agda

```diff
@@ -32,9 +47,28 @@ prime? 0             = no λ()
 prime? 1             = no λ()
 prime? (suc (suc n)) = all? (λ _ → ¬? (_ ∣? _))
 
+-- Relation between compositeness and primality
+
+composite⇒¬prime : ∀ n → Composite n → ¬ Prime n
```

Comment:
Can the `n` be made implicit in these 4 proofs or does the use of `⊥` prevent that?

Reply:
I think it should be fine! All of these proofs have _something_ positive, either a proof of compositeness or primality or a proof of being greater than or equal to 2, that can determine n, as an argument.